test/specs/module.ts

```diff
@@ -1,5 +1,6 @@
 import { tsdJsdocTestCase } from '../lib';
 
 suite('Module Checks', () => {
-    tsdJsdocTestCase('All', 'module_all');
+		tsdJsdocTestCase('All', 'module_all');
+		tsdJsdocTestCase('Quoted', 'module_quoted');
```

Comment:
Tabs here too, should be 4 spaces.

Reply:
Not sure what's going on with my IDE today. Sorry!